tests/LoadablePieces/LoadablePieces.cpp

```diff
@@ -0,0 +1,52 @@
+
+// LoadablePieces.cpp
+
+// Implements the LoadablePieces test main entrypoint
+
+#include "Globals.h"
+#ifdef _WIN32
+	#include <direct.h>
+	#define GetCurrentFolder _getcwd
+#else
+	#include <unistd.h>
+	#define GetCurrentFolder getcwd
+#endif
+#include "Generating/PrefabPiecePool.h"
+
+
+
+
+
+static int DoTest(void)
+{
+	cPrefabPiecePool test;
+	auto res = test.LoadFromFile("Test.cubeset", true);
+	if (!res)
+	{
+		LOGWARNING("Loading from file \"Test.cubeset\" failed.");
+		return 1;
+	}
+	LOG("Loaded %u regular pieces and %u starting pieces", static_cast<unsigned>(test.GetAllPiecesCount()), static_cast<unsigned>(test.GetStartingPiecesCount()));
```

Comment:
It might be worth putting an assert on these values, to check it loaded everything.